pkg/healthcheck/handler.go

```diff
@@ -94,5 +136,8 @@ func (hc *HealthCheck) Get() Status {
 
 // Ready is a shortcut for Set(Ready) (kept for backwards compatibility)
 func (hc *HealthCheck) Ready() {
+	hc.upTimeStats = upTimeStats{
+		StartedAt: time.Now(),
```

Comment:
The [process' start time](https://godoc.org/github.com/shirou/gopsutil/process#Process.CreateTime) is recorded as a metric:

```
# HELP process_start_time_seconds Start time of the process since unix epoch in seconds.
# TYPE process_start_time_seconds gauge
process_start_time_seconds 1.55480578686e+09
```

Should we add this one as well?

Reply:
Should I add this as well?

Reply:
I meant the other way around, to add the "Started At" as a metric.

Reply:
Where should I add it?

Reply:
Looks like the handler isn't receiving a metrics object. Perhaps it's better to open a follow-up issue for now.

Reply:
Perhaps a follow-up issue of what should be done would be more helpful, as on my local build the `/metrics` endpoint does not return any `process` information. :)